internal/restic/snapshot_policy.go

```diff
@@ -90,7 +90,8 @@ func ApplyPolicy(list Snapshots, p ExpirePolicy) (keep, remove Snapshots) {
 		{p.Yearly, y, -1},
 	}
 
-	for _, cur := range list {
+	for nr, cur := range list {
+		nr = nr + 1
```

Comment:
hm, now `nr` is always incremented, so for the first element of `list`, `nr` is set to `1`. Was that intentional?

Reply:
You are right, we dont need this line